.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+### Motivation
```

Comment:
@normanmaurer - I'll just stay consistent with the wiki ... we can update later if we want.